ARROW-4444: [Testing] Add CSV test file for use by DataFusion unit tests and examples

andygrove opened the pull request.

xhochy commented on data/csv/uk_cities.csv:
```diff
@@ -0,0 +1,37 @@
+"Elgin, Scotland, the UK",57.653484,-3.335724
```
Where does this file originate from?

andygrove replied:
I'm trying to remember ... if I can't find it, I will download fresh from here:

https://simplemaps.com/data/world-cities

The free download here uses the Creative Commons Attribution License.

xhochy replied:
Can you write a LICENSE file and remark that in there?

andygrove replied:
@xhochy For now I removed the files that are not used, and I manually created a new uk_cities.csv with just two cities and looked up lat/lon online, so this is safe to merge.

Once the DataFusion donation is accepted I'll create some better test files and delete the trivial ones. I don't want to complicate the donation by doing that now and updating all the tests.

andygrove commented:
@xhochy I deleted all the files and started again. There is now a single generated file. I have updated my DataFusion tests and examples to use this file now.
